testing/features/support/drivers/local.rb

```diff
@@ -20,13 +20,13 @@ def register
     def service
       return unless safari?
 
-      #Selenium::WebDriver::Safari.technology_preview!
+      # Selenium::WebDriver::Safari.technology_preview!
       Selenium::WebDriver::Service.safari({ args: ["--diagnose"] })
     end
 
     def desired_capabilities
       Selenium::WebDriver::Remote::Capabilities.new.tap do |capabilities|
-        if false
+        if false # rubocop:disable Lint/LiteralAsCondition
```

Comment:
`if false` will never run, so should this code even exist?

Reply:
Not sure about this.... I remember some "debug" code we did where we set this, but I swore we never merged it in. Give me an hour and I'll chase this up.